src/recognizer/recognizer.py

```diff
@@ -78,6 +80,9 @@ def run_detector(detector, path):
 
         result = {key:value.numpy() for key,value in result.items()}
 
+        logger.info('Found {0} objects!'.format(len(result["detection_scores"])))
+        logger.info('Inference time: {0}'.format(end_time - start_time))
```

Comment:
Log should be a debug log via `logger.debug()`, as we will want to hide this information when not in a `--verbose` configuration. Also, "f-strings" are preferred over `.format()`. So this would be:

```py
logger.debug(f'Found {len(result["detection_scores"]))} objects!'
logger.debug(f'Inference time: {end_time - start_time}')
```

Could also make a variable for the object count to keep it a little more clean.